include/exadg/operators/operator_base.cpp

```diff
@@ -823,10 +861,10 @@ OperatorBase<dim, Number, n_components>::internal_calculate_system_matrix(
   SparseMatrix & system_matrix) const
 {
   // assemble matrix locally on each process
-  if(evaluate_face_integrals() and is_dg)
+  if(evaluate_face_integrals())
```

Comment:
I find this logic irritating or not consistent with other parts of `OperatorBase` where we pass `empty` functions to `matrix_free->loop()`. I think we should distinguish between `dg` and `cg` on the top level, and inside each variant introduce another if depending on whether we have face loops or not. If we enter a `face_loop`, we will then also do the face integrals. See also comment https://github.com/exadg/exadg/pull/555/files#r1457261288